### ISSUE TITLE
coverage --help is not correct: `=` is missing for HTML 

### ISSUE THREAD

**Issue by JochenHiller** [CodeIntelligenceTesting/cifuzz#811]
Using `cifuzz coverage --help` shows how to generate HTML only:

```
HTML
    cifuzz coverage --output coverage-report <fuzz test>
```

But the text was missing an equal: correct is `--output=coverage-report`

**Comment by JochenHiller** [comment]
This PR will fix it: https://github.com/CodeIntelligenceTesting/cifuzz/pull/810

**Comment by JochenHiller** [comment]
I checked it again: did work with and without `=` sign.
So it is more a question to be consistent between help and not that something is not working.

**Comment by JochenHiller** [comment]
My fault:

I did run coverage with wrong options `-output report` which did not work (due to missing `-`).
Then I changed to  `-output=report` which did work, but folder name was `utput=report`. I just saw that later when checking coverage results.

When using correct option either `--output` or `-o` it does work as described.

Sorry for inconvenience, next time I will test more thoroughly first.
I will close PR as well.
